src/Transform/ONNX/ConstProp.cpp

```diff
@@ -392,6 +392,16 @@ template <typename T>
 struct ElementWiseUnaryOpImpl<ONNXSqrtOp, T> {
   static T impl(T val) { return sqrt(val); }
 };
+
+template <typename T>
+struct ElementWiseUnaryOpImpl<ONNXReluOp, T> {
+  static T impl(T val) {
+    if (val < 0)
+      return 0;
+    return val;
+  }
+ };
```

Comment:
Remove spaces before `};` too, which caused clang-format check fail.

Reply:
It's generally a good idea to do
```
clang-format-9 -i filename
```
for any .h .hpp .c .cpp files you modified so that they get formatted according to the clang-format check.